cookie.js

```diff
@@ -21,10 +21,14 @@ export function setCookie (key, value, options = {})
 {
     options = merge({
         path: "/",
-        secure: window.location.protocol === "https",
         expires: 30,
     }, options);
 
+    if (window.location.protocol === "https")
+    {
+        options.secure = true;
+    }
```

Comment:
That is the wrong place to fix this, as any other lib maybe also would like to pass `false` and have this option ignored.

Instead you should add it here: https://github.com/Becklyn/mojave/pull/88/files#diff-c1a140ec34b6072d83f809ef70e9baf4R119

```js
for (let optionName in options)
{   
    const optionValue = options[optionName];

    // ignore `false` values
    if (!options.hasOwnProperty(optionName) || optionValue === false)
    {
        continue;
    }

    // ...
}
```

Reply:
You're absolutely right. I've updated the PR.